projects/components/src/back-button/back-button.component.scss

```diff
@@ -0,0 +1,16 @@
+@import 'color-palette';
+@import 'font';
+
+.back {
+  display: flex;
+  height: 26px;
+  width: 64px;
+  align-items: center;
+  color: $color-text-emphasis;
+  cursor: pointer;
+
+  .label {
+    @include body-2-medium($color-text-emphasis);
+    margin-left: 8px;
```

Comment:
nit: can we use `gap` at L11 instead of margin here?

Reply:
Updated

Reply:
Used `ht-button`, removed styles.